theseus_gui/src/components/ui/AppBar.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { useRoute, useRouter } from 'vue-router'
+import { ChevronLeftIcon, ChevronRightIcon } from 'omorphia'
+
+const route = useRoute()
+const router = useRouter()
+</script>
+
+<template>
+  <div class="appbar">
+    <section class="navigation-controls">
+      <ChevronLeftIcon @click="router.back()" />
+      <ChevronRightIcon @click="router.forward()" />
+      <p>{{ route.name }}</p>
+    </section>
+    <section class="mod-stats">
+      <p>Updating 2 mods...</p>
+      <p>123 mods installed</p>
+    </section>
```

Comment:
The style here is generally pretty off from the design. The text is too big, the arrows are sized incorrectly, there isn't enough padding in the app bar- etc